### PR TITLE
Fixing doco for ShellElements.md

### PR DESCRIPTION
**Pull request by andrsd**
- fixing TeX typos
- fixing Tex notation of an author name

### CODE REVIEW

File: modules/tensor_mechanics/doc/content/bib/tensor_mechanics.bib [hunk 1]
```diff
@@ -683,7 +683,7 @@ @article{Mikelic2019
 }
 
 @phdthesis{Bolourchi1979,
-Author = {Bolourchi, Sa{\''i}d},
+Author = {Bolourchi, Sa{\"\i}d},
```
Comment (recuero) [inline modules/tensor_mechanics/doc/content/bib/tensor_mechanics.bib:686]: Would `{\"i}` work? Instead of `{\''\i}`

**Review by recuero — DISMISSED** [review]

File: modules/tensor_mechanics/doc/content/bib/tensor_mechanics.bib (hunk 1 above)
Comment (andrsd) [reply modules/tensor_mechanics/doc/content/bib/tensor_mechanics.bib:686]: no, you would get 3 dots. `\i` removes the dot from `i`.

Comment (andrsd) [reply modules/tensor_mechanics/doc/content/bib/tensor_mechanics.bib:686]: hold on, maybe it would... just trying it... this used to be a thing...

Comment (andrsd) [reply modules/tensor_mechanics/doc/content/bib/tensor_mechanics.bib:686]: ok, it is no longer a thing. `\"i` works too.

**Comment by andrsd** [comment]
@recuero Does github allow you to approve the pull request? Or can you just leave a comment? If so, then I need to pull in somebody with more github power, so we can merge this.

**Comment by recuero** [comment]
@andrsd You need someone with real power, I can't merge

**Review by recuero — APPROVED** [review]

**Comment by andrsd** [comment]
> You need someone with real power, I can't merge

Review and merge are 2 different things. We need at least one approving review for merging. And it looks that even your review is not enough to get an approving one...

I will ping someone.  Thanks!

**Review by loganharbour — APPROVED** [review]

**Review by permcody — APPROVED** [review]

**Comment by permcody** [comment]
Yes, two approvals!

**Comment by andrsd** [comment]
Thanks guys!

**Pull request merged**